src/web/libdoc/i18n/translations.json

```diff
@@ -56,5 +56,34 @@
     "generatedBy": "Luotu",
     "on": "",
     "chooseLanguage": "Valitse kieli"
+  },
+  "nl": {
+    "code": "nl",
+    "intro": "Introductie",
+    "libVersion": "Bibliotheek versie",
+    "libScope": "Bibliotheek bereik",
+    "importing": "Importeren",
+    "arguments": "Parameters",
+    "doc": "Documentatie",
+    "keywords": "Actiewoorden",
+    "tags": "Labels",
+    "returnType": "Andwoord type",
+    "kwLink": "Actiewoord link",
```

Comment:
_Verwijzing naar actiewoord_ of _koppeling naar actiewoord_ als je het zou willen vertalen, maar linken wordt tegenwoordig ook wel als Nederlands woord beschouwd in het webdomein. Dna wordt het _Link naar actiewoord_ of _actiewoordlink_.

Reply:
`Link naar actiewoord` van gemaakt